index.ts

```diff
@@ -20,13 +20,19 @@ export interface FeatureData {
     usage_stats?: usage_stats_url | [usage_stats_url, usage_stats_url, ...usage_stats_url[]];  // A single URL or an array of two or more
 }
 
-type browserIdentifier = "chrome" | "edge" | "firefox" | "safari";
+type browserIdentifier = "chrome" | "chrome_android" | "edge" | "firefox" | "firefox_android" | "safari" | "safari_ios";
+
+type BaselineHighLow = "high" | "low";
 
 interface SupportStatus {
-    /** Whether the feature achieved baseline status */
-    is_baseline: boolean;
-    /** Date the feature achieved baseline status */
+    /** Whether the feature is Baseline (low substatus), Baseline (high substatus), or not (false) */
+    baseline?: BaselineHighLow | false;
+    /** Whether the feature is Baseline (legacy) */
+    is_baseline?: true | false;
```

Comment:
Should this be removed from the schema? Or are we keeping it for backwards compatibility? (No need to do so for MDN)

Reply:
I can't remove all of the old statuses yet—they need actual reviews for the feature lists. I'll do that in a follow up PR.